drawee/src/main/java/com/facebook/drawee/drawable/ScalingUtils.java

```diff
@@ -415,10 +427,16 @@ public Matrix getTransform(
         int childHeight,
         float focusX,
         float focusY) {
-      mScaleTypeFrom.getTransform(transform, parentBounds, childWidth, childHeight, focusX, focusY);
-      transform.getValues(mMatrixValuesFrom);
-      mScaleTypeTo.getTransform(transform, parentBounds, childWidth, childHeight, focusX, focusY);
-      transform.getValues(mMatrixValuesTo);
+      if (mMatrixValuesFrom == null) {
+        mMatrixValuesFrom = new float[9];
+        mScaleTypeFrom.getTransform(transform, mBoundsFrom, childWidth, childHeight, focusX, focusY);
```

Comment:
Let's use `parentBounds` in case `mBoundsFrom` is null. Same for `mBoundsTo` below.

`Rect boundsFrom = (mBoundsFrom != null) ? mBoundsFrom : parentBounds;`
`Rect boundsTo = (mBoundsTo != null) ? mBoundsTo : parentBounds;`